aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -53,18 +53,19 @@ static void apply_batched_inverse_lib(Tensor& self, Tensor& self_inv, Tensor& in
   // heuristic:
   //   cublas_x_batched doesn't work very well for small batchsize
   //   cublas_x_batched is intended to be used for matrices of small sizes where the launch overhead is a significant factor.
-  const bool use_cusolver_parallel_launch = (batch_size <= 8) || (/* batch_size > 8 && */ n >= 512);
+  const bool use_cusolver_for_loop = (batch_size <= 8) || (/* batch_size > 8 && */ n >= 512);
```

Comment:
Do we use this variable anywhere else? If not, maybe we can put the boolean logic directly in the if statement to avoid confusion with the unconventional naming. We could extend the comment to explain that for the cases listed (small batchsize and/or large matrices) we iterate over the batches and call the non-batched routine.